Editor/NodeReflection.cs

```diff
@@ -111,6 +112,10 @@ public class NodeReflectionData
         /// Can this node be deleted from the graph 
         /// </summary>
         public bool Deletable { get; set; }
+        /// <summary>
+        /// Can this node be moved in from graph
+        /// </summary>
+        public bool MoveAble { get; set; }

```

Comment:
Could you change this to `Moveable` to be consistent with Unity's `Capabilities.Moveable`